Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetConnection.tsx

```diff
@@ -232,10 +251,13 @@ function AssetConnectionWindow(props: { Name: string, ID: number, TypeID: number
     return (
         <div className="card" style={{ flex: 1, display: 'flex', flexDirection: 'column', overflow: 'hidden' }}>
             <div className="card-header">
-                <div className="row">
-                    <div className="col">
+                <div className="row justify-content-between">
+                    <div className="col-6">
                         <h4>Connections:</h4>
                     </div>
+                    <div className="pr-4">
+                        <LocationDrawingsButton Locations={locations} />
```

Comment:
Can we feed an external loading status into this location drawing button? That way we can have this be loading while we wait for the server call.